cmake/modules/BuildRocksDB.cmake

```diff
@@ -74,7 +87,8 @@ function(build_rocksdb)
     BUILD_COMMAND "${make_cmd}"
     BUILD_ALWAYS TRUE
     BUILD_BYPRODUCTS "${rocksdb_LIBRARY}"
-    INSTALL_COMMAND "true")
+    INSTALL_COMMAND "true"
+    LIST_SEPARATOR !)
```

Comment:
i'd prefer moving the change related to separator in BuildRocksDB.cmake into a separated commit.